### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by CharlesChen0823**
because my current person branch has many different, so specify current modify for which test is difficult.

### CODE REVIEW

File: src/html5/parser.rs [hunk 1]
```diff
@@ -112,17 +112,6 @@ macro_rules! current_node {
     }};
 }
 
-macro_rules! current_node_mut {
```
Comment (jaytaph) [inline src/html5/parser.rs:115]: ah, this is nice.. there was only one place that still uses this.. and we cannot get rid of ALL the macro's (yet).. but the more we can remove, the better

File: src/html5/parser.rs [hunk 2]
```diff
@@ -2817,6 +2805,9 @@ impl<'stream> Html5Parser<'stream> {
                 self.parse_error("doctype not allowed in before head insertion mode");
                 // ignore token
             }
+            Token::StartTagToken { name, .. } if name == "html" => {
```
Comment (jaytaph) [inline src/html5/parser.rs:2808]: does this fix a test or did you just happen to find some unimplemented start-tag?

Comment (CharlesChen0823) [reply src/html5/parser.rs:2808]: in testing find this issue, and then reading docs, found `html` start-tag not implemented.

File: src/html5/parser.rs (hunk 1 above)
Comment (CharlesChen0823) [reply src/html5/parser.rs:115]: because found an issue in `parser.rs`, which is using the macro, then fixed that issue, the macro is not used, so delete.

**Comment by jaytaph** [comment]
@CharlesChen0823 thanks for the pr

**Pull request merged**